Examples/apps/MacExample/MacExample/AppDelegate.swift

```diff
@@ -22,7 +22,7 @@ class AppDelegate: NSObject, NSApplicationDelegate {
             .flushInterval(10)
         
         analytics = Analytics(configuration: configuration)
-
```

Comment:
Should we maybe add a comment: `// Uncomment to add notification tracking to your app` or something?

Reply:
It's a placeholder until I get full lifecycle stuff on Mac working.